packages/velo-external-db/test/e2e/app_data.e2e.spec.js

```diff
@@ -26,7 +26,7 @@ describe(`Velo External DB Data REST API: ${currentDbImplementationName()}`,  ()
         await dbTeardown()
     }, 20000)
 
-    test('find api', async() => testIfSchemaSupportsFindWithSort(env, async() => {
+    testIfSchemaSupportsFindWithSort(env)('find api', async() => {
```

Comment:
after exposing the supportedOperations const, you can pass it here instead the all env.